src/bin/ios.rs

```diff
@@ -0,0 +1,14 @@
+//! This is an example of how to shutdown the Wii using Luma.
+
+#![no_std]
+
+extern crate luma_core;
+extern crate luma_runtime;
+
+use luma_core::ios;
+
+fn main() {
+    let fd = ios::open("/dev/stm/immediate\0", ios::Mode::None).unwrap();
+    ios::ioctl(fd, 0x2003, &[], &[]).unwrap();
```

Comment:
This is hard to read, maybe move `0x2003` to a named const eg. `const SOME_IOCTL: u32 = 0x2003`